portal/templates/initial_queries_macros.html

```diff
@@ -8,31 +8,99 @@ <h4 class="tnth-headline">{{ _("Terms of Use") }}</h4>
+                          <span class="terms-tick-box-text">&nbsp;By checking this box, I confirm that I have read and accept the website <a
+                            href="{{privacy_url}}" class="required-link">privacy policy</a> and <a
+                            href="{{terms_url}}" class="required-link">terms</a></span>
+                          <input type="hidden"
+                            data-type="terms"
+                            data-agree="false"
+                            data-core-data-type="privacy_policy"
+                            data-tou-type="privacy policy"
+                            data-url="{{privacy_url}}"/>
+                          <input type="hidden"
+                            data-agree="false"
+                            data-type="terms"
+                            data-tou-type="website terms of use"
+                            data-core-data-type="website_terms_of_use"
+                            data-url="{{tou_url}}" />
+                        {% endtrans %}
+                    </label>
+                {% else %}
+                    <!-- EPROMS - user account has been created by staff -->
+                    {% if user.has_role(ROLE.PATIENT) %}
+                         <label
+                          class="terms-label tnth-hide"
+                          data-agree="false"
+                          data-type="terms"
+                          data-tou-type="subject website consent"
+                          data-core-data-type="subject_website_consent"
+                          data-url="{{tou_url}}">
+                          <i class="fa fa-square-o fa-lg terms-tick-box edit-view"></i>
+                          <span class="edit-view"><span class="terms-tick-box-text">{{_("By checking this box, I confirm that I have read the information notice above and consent and agree to the processing of my personal information (including my health information) on the terms described in this Consent.") }}</span></span>
+                          <div class="display-view text-warning tnth-hide">{{_("You have previously provided your consent to the website during a visit at your treating site. If you would like a copy of this please contact your study contact.")}}<br/></div>
+                        </label>
+                         <label class="terms-label tnth-hide">
+                            <i class="fa fa-square-o fa-lg terms-tick-box"></i>
+                            {% trans privacy_url=url_for('portal.privacy', disableLinks="true"), terms_url=url_for('portal.terms_and_conditions', disableLinks="true") %}
+                              <span class="terms-tick-box-text">&nbsp;By checking this box, I confirm that I have read and accept the website <a href="{{privacy_url}}" class="required-link">privacy policy</a> and
+                                <a href="{{terms_url}}" class="required-link">terms</a></span>
```

Comment:
Same issue as above, I don't *think* there's any translatable text below this link